packages/wrangler/src/assets.ts

```
@@ -287,15 +285,6 @@ const buildAssetManifest = async (dir: string) => {
 					assetsIgnoreFilePresent
 				);
 
-				if (counter >= MAX_ASSET_COUNT) {
```

We wan the API to handle this

---

why not fail early tho?

---

what do you mean? the api handles the limit, this will fail on that api call.

---

Because we don't know what the limit is and will be. For some users it'll be 20k, for some 100k, some in the future may have 200k.

Wrangler should really _never_ hardcode limits, they can always change. The API is the source of truth.

I suspect we'll have a /limits endpoint in the future which we can pull from but for now, we'll do this.

---

cool cool! got it! thx Walshy 🙏